finetuner/tuner/callback/evaluation.py

```diff
@@ -0,0 +1,132 @@
+import math
+from typing import TYPE_CHECKING, Optional
+
+from ... import embed
+from ..dataset.helper import batch_document_sequence
+from ..evaluation import Evaluator
+from .base import BaseCallback
+
+if TYPE_CHECKING:
+    from ...helper import DocumentSequence
+    from ..base import BaseTuner
+
+
+class Evaluation(BaseCallback):
```

Comment:
i suggest we give a more precise naming, to clarify when use it standalone or as callback, such as name it `EvaluationCallback`.